v2/api/apimanagement/v1api20220801/api_types_gen.go

```diff
@@ -905,12 +890,7 @@ func (api *Api_Spec) AssignProperties_From_Api_Spec(source *storage.Api_Spec) er
 	}
 
 	// ApiVersionDescription
-	if source.ApiVersionDescription != nil {
-		apiVersionDescription := *source.ApiVersionDescription
-		api.ApiVersionDescription = &apiVersionDescription
-	} else {
-		api.ApiVersionDescription = nil
-	}
+	api.ApiVersionDescription = genruntime.ClonePointerToString(source.ApiVersionDescription)
```

Comment:
We could probably write a more generic `ClonePtr` function, using generics now.
```
func ClonePointer[T any](ptr *T) *T {
	if ptr == nil {
		return nil
	}

	result := *ptr
	return &result
}
```

Reply:
Good idea for a followup PR.